app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  scope :update_post, ->(id, text) { find_by(id:).update(text:) }
+
+  def most_recent_comments
+    comments.order(created_at: :desc).first(5)
+  end
+end
```

Comment:
- [ ] Kindly add the post counter method. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=A%20method%20that%20updates%20the%20posts%20counter%20for%20a%20user.)